proto/desmos/subspaces/v1/genesis.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+package desmos.subspaces.v1;
+
+import "gogoproto/gogo.proto";
+import "desmos/subspaces/v1/subspace.proto";
+
+option go_package = "github.com/desmos-labs/desmos/v2/x/subspaces/types";
+
+// GenesisState contains the data of the genesis state for the subspaces module
+message GenesisState {
+  option (gogoproto.equal) = true;
+  option (gogoproto.goproto_stringer) = true;
+
+  uint64 initial_subspace_id = 1
+      [ (gogoproto.customname) = "InitialSubspaceID" ];
+
+  repeated desmos.subspaces.v1.Subspace subspaces = 2
+      [ (gogoproto.nullable) = false ];
+
+  repeated ACLEntry acl = 3
```

Comment:
I've used the ACL term since it's the acronym for Access Control List. I think adding _entries_ (and making it _Access Control List entries_) would be redundant